### PR TITLE
Adds aria-label attrs to img elements

### PR DESCRIPTION
**Pull request by mabashian**
##### SUMMARY
See: https://dequeuniversity.com/rules/axe/3.5/image-alt?application=AxeChrome

##### ISSUE TYPE
 - Bugfix Pull Request

##### COMPONENT NAME
 - UI

### CODE REVIEW

File: awx/ui/client/installing.template.ejs [hunk 1]
```diff
@@ -22,7 +22,11 @@
   </head>
   <body>
     <div class="at-Upgrade--panel">
-      <img src="{% static 'assets/logo-header.svg' %}" width="200"/>
+      <img
+        ng-attr-aria-label="{{ BRAND_NAME === 'Tower' ? 'Ansible Tower Logo' : 'Ansible AWX Logo' }}"
```
Comment (jakemcdermott) [inline awx/ui/client/installing.template.ejs]: should we mark these for translation?

Comment (mabashian) [reply awx/ui/client/installing.template.ejs]: Yep - pushed another commit to do this

File: awx/ui/client/lib/components/layout/layout.partial.html [hunk 2]
```diff
@@ -2,7 +2,7 @@
     <div class="at-Layout-topNav">
         <at-top-nav-item is-shown="missingLicense" class="at-Layout-topNavItem--logo">
             <a href="/#/">
-                <img ng-src="/static/assets/logo-header.svg">
+                <img ng-attr-aria-label="{{ $root.BRAND_NAME === 'Tower' ? $root.appStrings.get('logos.TOWER_LOGO') : $root.appStrings.get('logos.AWX_LOGO') }}" ng-src="/static/assets/logo-header.svg" />
```
Comment (jakemcdermott) [inline awx/ui/client/lib/components/layout/layout.partial.html:5]: Screenshot from 2020-03-11 15-36-07

:+1:

**Review by jakemcdermott — APPROVED** [review]
I tested this by changing [this code](https://github.com/ansible/awx/blob/devel/awx/main/middleware.py#L229) to:

```python
class MigrationRanCheckMiddleware(MiddlewareMixin):
    def process_request(self, request):
        executor = MigrationExecutor(connection)
        plan = executor.migration_plan(executor.loader.graph.leaf_nodes())
        if True and \
                getattr(resolve(request.path), 'url_name', '') != 'migrations_notran':
            return redirect(reverse("ui:migrations_notran"))
```

.. and then inspecting the rendered html on page load:

Screenshot from 2020-03-11 15-36-07

**Pull request merged**